<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Menu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_welcome">
     <property name="text">
      <string>Welcome to Sleep Tracker</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_daily_entry">
     <property name="text">
      <string>Fill Daily Entry</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_monthly_analysis">
     <property name="text">
      <string>View Monthly Analysis</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_treatment_report">
     <property name="text">
      <string>End Treatment &amp; Generate Report</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_logout">
     <property name="text">
      <string>Log Out</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
